src/components/ebay-breadcrumb/README.md

```diff
@@ -0,0 +1,40 @@
+# ebay-breadcrumb
+
+## ebay-breadcrumb Usage
+### Last item as the parent
+```marko
+<ebay-breadcrumb aria-label='You are here'>
+    <ebay-breadcrumb-item href='https://...'>eBay</ebay-breadcrumb-item>
+    <ebay-breadcrumb-item href='https://...'>Auto Parts and Vehicles</ebay-breadcrumb-item>
+    <ebay-breadcrumb-item href='https://...' _sp='' navsrc='{}' navsrc='{}'>Motors Parts and Accessories</ebay-breadcrumb-item>
+</ebay-breadcrumb>
+```
+### Last item as the current
```

Comment:
I think the `README.md` should just have the simplest usage example (maybe this one), and leave example code in general for the `examples/` section.